Mathlib/Order/Filter/Basic.lean

```diff
@@ -1076,6 +1076,13 @@ theorem eventuallyEq_iff_all_subsets {f g : α → β} {l : Filter α} :
     f =ᶠ[l] g ↔ ∀ s : Set α, ∀ᶠ x in l, x ∈ s → f x = g x :=
   eventually_iff_all_subsets
 
+protected theorem _root_.Filter.EventuallyEq.fun_star {R : Type*} [Star R] {f g : α → R}
```

Comment:
Aren't you already in the Filter namespace? So
```suggestion
protected theorem EventuallyEq.fun_star {R : Type*} [Star R] {f g : α → R}
```
or arguably
```suggestion
protected theorem EventuallyEq.fun_star [Star β] {f g : α → β}
```
to match the style above.